raiden_contracts/tests/test_contracts_compilation.py

```diff
@@ -1,17 +1,21 @@
 from pathlib import Path
-
 import pytest
 
 from raiden_contracts.contract_manager import (
     ContractManager,
     CONTRACTS_SOURCE_DIRS,
+    CONTRACTS_PRECOMPILED_PATH,
 )
 from raiden_contracts.constants import (
     CONTRACT_TOKEN_NETWORK,
     ChannelEvent,
 )
 
 
+def test_verification():
+    ContractManager.verify_contracts(CONTRACTS_SOURCE_DIRS, CONTRACTS_PRECOMPILED_PATH)
```

Comment:
This is not testing much. Could you please add test to ensure that it raises when the contract checksum doesn't match? This is the case we are interest in

Reply:
Argh, I tested this in the terminal, but I forgot to write a test. Thank you.